src/api/resources.js

```diff
@@ -18,6 +19,7 @@ export const apiResources = {
   DEPLOYMENT: deployment,
   DEPLOYMENTS: deployments,
   ENVIRONMENT: environments,
+  ENVVARS: environmentVariables,
```

Comment:
Can be removed if redux state for envvars are not used.

Reply:
Deleted.